apps/site/pages/en/learn/modules/how-to-use-streams.md

```diff
@@ -0,0 +1,599 @@
+[`TextDecoderStream`]: https://developer.mozilla.org/en-US/docs/Web/API/TextDecoderStream
+[`EventEmitter`]: https://nodejs.org/api/events.html#class-eventemitter
+[`Writable`]: https://nodejs.org/api/stream.html#stream_writable_streams
+[`Readable`]: https://nodejs.org/api/stream.html#stream_readable_streams
+[`Duplex`]: https://nodejs.org/api/stream.html#stream_duplex_and_transform_streams
+[`Transform`]: https://nodejs.org/api/stream.html#stream_duplex_and_transform_streams
+[`drain`]: https://nodejs.org/api/stream.html#stream_event_drain
+[`on('data')`]: https://nodejs.org/api/stream.html#stream_event_data
+[`data`]: https://nodejs.org/api/stream.html#stream_event_data
+[`on('end')`]: https://nodejs.org/api/stream.html#event-end
+[`end`]: https://nodejs.org/api/stream.html#event-end
+[`on('readable')`]: https://nodejs.org/api/stream.html#event-readable
+[`on('close')`]: https://nodejs.org/api/stream.html#event-close_1
+[`on('error')`]: https://nodejs.org/api/stream.html#event-error_1
+[`.read()`]: https://nodejs.org/docs/latest/api/stream.html#stream_readable_read_size
+[`.write()`]: https://nodejs.org/api/stream.html#stream_writable_write_chunk_encoding_callback
+[`_write`]: https://nodejs.org/api/stream.html#writable_writechunk-encoding-callback
+[`.end()`]: https://nodejs.org/api/stream.html#writableendchunk-encoding-callback
+[`_transform`]: https://nodejs.org/api/stream.html#transform_transformchunk-encoding-callback
+[`Readable.from()`]: https://nodejs.org/api/stream.html#streamreadablefromiterable-options
+[`highWaterMark`]: https://nodejs.org/api/stream.html#stream_buffering
+[`.pipe()`]: https://nodejs.org/docs/latest/api/stream.html#stream_readable_pipe_destination_options
+[`pipeline()`]: https://nodejs.org/api/stream.html#stream_stream_pipeline_streams_callback
+[`async pipeline()`]: https://nodejs.org/api/stream.html#streampipelinesource-transforms-destination-options
+[`Web Streams`]: https://nodejs.org/api/webstreams.html
+[`ReadableStream`]: https://nodejs.org/api/webstreams.html#class-readablestream
+[`WritableStream`]: https://nodejs.org/api/webstreams.html#class-writablestream
+[`TransformStream`]: https://nodejs.org/api/webstreams.html#class-transformstream
+[`WHATWG Streams Standard`]: https://streams.spec.whatwg.org/
+[`backpressure guide`]: /learn/modules/backpressuring-in-streams
+[`fs.readStream`]: https://nodejs.org/api/fs.html#class-fsreadstream
+[`http.IncomingMessage`]: https://nodejs.org/api/http.html#class-httpincomingmessage
+[`process.stdin`]: https://nodejs.org/api/process.html#processstdin
+[`fs.WriteStream`]: https://nodejs.org/api/fs.html#class-fswritestream
+[`process.stdout`]: https://nodejs.org/api/process.html#processstdout
+[`process.stderr`]: https://nodejs.org/api/process.html#processstderr
+[Matteo Collina]: https://github.com/mcollina
+[Platformatic's Blog]: https://blog.platformatic.dev/a-guide-to-reading-and-writing-nodejs-streams
```

Comment:
Some links are inline, while others are like this, can you make them all uniform?

Reply:
The rule that should be applied is this. If the link is used once, we put it inline if it is used several times in the bottom

Reply:
That's not what happening in this article. In this article, some are inline, some aren't. IMO one of the following should be done
1. All inline (preferably not)
2. All text-based references (I  like this the best)
3. Inline unless used multiple times

Reply:
I think (almost) all of them are used only once. I like the idea of having them all at the bottom as it's like a bibliography. I haven't seen anything documented about this rule, but I'm happy to go either way as long as we agree

Reply:
IMO as you said, having them at the bottom is cleaner

Reply:
I put them all at the end, hope that's okay